modules/local/ascc/merge_tables/main.nf

```diff
@@ -30,18 +32,20 @@ process ASCC_MERGE_TABLES {
     task.ext.when == null || task.ext.when
 
     script:
-    def prefix                      = task.ext.prefix           ?: "${meta.id}"
-    def args                        = task.ext.args             ?: ""
-    def coverage                    = coverage                  ? "-c ${coverage}"                  : ""
-    def tiara                       = tiara                     ? "-t ${tiara}"                     : ""
-    def nt_kraken                   = nt_kraken                 ? "-nk ${nt_kraken}"                : ""
-    def nt_blast                    = nt_blast                  ? "-nb ${nt_blast}"                 : ""
-    def dim_reduction_embeddings    = dim_reduction_embeddings  ? "-dr ${dim_reduction_embeddings}" : ""
-    def nr_diamond                  = nr_diamond                ? "-nd ${nr_diamond}"               : ""
-    def uniprot_diamond             = uniprot_diamond           ? "-ud ${uniprot_diamond}"          : ""
-    def btk                         = btk                       ? "-btk ${btk}"                     : ""
-    def btk_busco                   = btk_busco                 ? "-bb ${btk_busco}"                : ""
-    def fcs_gx                      = fcs_gx                    ? "-fg ${fcs_gx}"                   : ""
+    def prefix                      = task.ext.prefix                           ?: "${meta.id}"
+    def args                        = task.ext.args                             ?: ""
+
+    def empty_file_size             = 80
```

Comment:
Is this because they are gzipped? I have noticed that empty gzipped files do sometimes vary in size. Would it nit be better to just pass in an empty list if files are empty?